### DISCUSSION TITLE
Rollback pod with high % of 4xx 5xx

### DISCUSSION THREAD

**Discussion by domgreen** [fluxcd/flux2#1080] [Q&A]
Hi,

I'm wondering if there is anything that allows us to rollback if there is a high percentage of errors in a deployed pod to the last previous healthy deploy.

For example, I rollout the pod fine, its health endpoint returns correctly and its running it just has 40% more errors of endpoints than the previous deploy. Is there a way to get this to automatically rollback? 

Alternatively, is there a traffic managing project that might be able to help here that would work well in conjunction with flux?

I have read the docs but maybe I was looking for the wrong thing.

Thanks

Dom

**Comment by hiddeco** [comment]
Sounds a lot like progressive delivery, for which we have [Flagger](https://github.com/fluxcd/flagger/)?
